akka-docs/rst/scala/code/docs/stream/cookbook/RecipeSimpleDrop.scala

```diff
@@ -15,11 +15,11 @@ class RecipeSimpleDrop extends RecipeSpec {
 
       //#simple-drop
       val droppyStream: Flow[Message, Message, NotUsed] =
-        Flow[Message].conflate(seed = identity)((lastMessage, newMessage) => newMessage)
+        Flow[Message].conflateWithSeed(seed = identity)((lastMessage, newMessage) => newMessage)
       //#simple-drop
       val latch = TestLatch(2)
       val realDroppyStream =
-        Flow[Message].conflate(seed = identity)((lastMessage, newMessage) => { latch.countDown(); newMessage })
+        Flow[Message].conflateWithSeed(seed = identity)((lastMessage, newMessage) => { latch.countDown(); newMessage })
```

Comment:
[Docs page](http://doc.akka.io/docs/akka/2.4.2-RC2/scala/stream/stream-cookbook.html#Dropping_elements) which includes this sample code mentions `conflate` a couple of times. It should now reference `conflateWithSeed` or the code should keep `conflate` and remove the `seed=identity` part.